modyn/storage/src/internal/file_wrapper/binary_file_wrapper.cpp

```diff
@@ -40,10 +37,13 @@ int64_t BinaryFileWrapper::get_label(uint64_t index) {
 
   get_stream()->seekg(static_cast<int64_t>(label_start), std::ios::beg);
 
-  std::vector<unsigned char> label_vec(label_size_);
-  get_stream()->read(reinterpret_cast<char*>(label_vec.data()), static_cast<int64_t>(label_size_));
+  std::vector<unsigned char> tmp_label_buffer(label_size_);
+  get_stream()->read(reinterpret_cast<char*>(tmp_label_buffer.data()), static_cast<int64_t>(label_size_));
 
-  return int_from_bytes(label_vec.data(), label_vec.data() + label_size_);
+  if (this->little_endian_) {
```

Comment:
No need for the `this` pointer (I guess it is a coding style question, but we never used it in the codebase)